compiler/rustc_resolve/src/imports.rs

```diff
@@ -1157,7 +1157,7 @@ impl<'a, 'b> ImportResolver<'a, 'b> {
                     }
                     _ => {
                         if !ident.is_path_segment_keyword() {
-                            format!("no `{}` external crate", ident)
+                            format!("no external crate `{}`", ident)
```

Comment:
I considered changing it to "no external crate **named**", but I didn't want to be wordy. What do you think?

```suggestion
                            format!("no external crate named `{}`", ident)
```

Reply:
I think that the PR's current wording is fine, but I don't feel strongly - either is an improvement.